packages/core/src/payment/strategies/googlepay/googlepay.ts

```diff
@@ -30,6 +55,11 @@ export type GooglePayCreator = BraintreeModuleCreator<GooglePayBraintreeSDK>;
 
 export interface GooglePayPaymentOptions {
     environment: EnvironmentType;
+    paymentDataCallbacks?: {
+        onPaymentDataChanged?(
```

Comment:
Does this need to be undefined?

Reply:
We use paymentDataCallbacks just in buyNowFlow. So in all other cases we don't need paymentDataCallbacks from GooglePayPaymentOptions interface

Reply:
Fixed